vpr/src/analytical_place/ap_netlist.h

```diff
@@ -0,0 +1,188 @@
+/**
+ * @file
+ * @author  Alex Singer
+ * @date    September 2024
+ * @brief   Defines the APNetlist class used to store the connectivity of
+ *          primitives in the Analytical Placement context.
+ *
+ * In the context of Analytical Placement, a block is a collection of atoms
```

Comment:
Will a block ever represent more than one cluster (i.e. grid location) for long carry chains etc.?  If not known we can be silent on this for now.

Reply:
This is not known. My gut feeling is to make it part of the block; but I would need to do more fiddling around.